supervisor/usb.h

```diff
@@ -35,6 +35,9 @@
 // it may be necessary to call it directly.
 void usb_background(void);
 
+// Schedule usb background
+void usb_background_schedule(void);
```

Comment:
this function name is the best I could come up, let me know if you want to rename it.